samples/bluetooth/conn_time_sync/src/timed_led_toggle.c

```diff
@@ -30,6 +30,15 @@ static struct gpio_dt_spec led = GPIO_DT_SPEC_GET_OR(DT_ALIAS(led1), gpios, {0})
 
 static uint8_t previous_led_value;
 
+struct gpio_nrfx_cfg {
```

Comment:
Discussed offline:
we should not mirror this type here. Instead, the required functionality could be added to the zephyr driver.